lib/ansiblelint/__main__.py

```diff
@@ -52,62 +52,64 @@ def main():
+    parser.add_argument('-p', dest='parseable',
+                        default=False,
+                        action='store_true',
+                        help="parseable output in the format of pep8")
+    parser.add_argument('--parseable-severity', dest='parseable_severity',
+                        default=False,
+                        action='store_true',
+                        help="parseable output including severity of rule")
+    parser.add_argument('-r', action='append', dest='rulesdir',
+                        default=[],
+                        help="specify one or more rules directories using "
+                             "one or more -r arguments. Any -r flags override "
+                             "the default rules in %s, unless -R is also used."
+                             % default_rulesdir)
+    parser.add_argument('-R', action='store_true',
+                        default=False,
+                        dest='use_default_rules',
+                        help="Use default rules in %s in addition to any extra "
+                             "rules directories specified with -r. There is "
+                             "no need to specify this if no -r flags are used"
+                             % default_rulesdir)
+    parser.add_argument('-t', dest='tags',
+                        action='append',
+                        default=[],
+                        help="only check rules whose id/tags match these values")
+    parser.add_argument('-T', dest='listtags', action='store_true',
+                        help="list all the tags")
+    parser.add_argument('-v', dest='verbosity', action='count',
+                        help="Increase verbosity level",
+                        default=0)
+    parser.add_argument('-x', dest='skip_list', default=[], action='append',
+                        help="only check rules whose id/tags do not " +
+                        "match these values")
+    parser.add_argument('--nocolor', dest='colored',
+                        default=hasattr(sys.stdout, 'isatty') and sys.stdout.isatty(),
+                        action='store_false',
+                        help="disable colored output")
+    parser.add_argument('--force-color', dest='colored',
+                        action='store_true',
+                        help="Try force colored output (relying on ansible's code)")
+    parser.add_argument('--exclude', dest='exclude_paths', action='append',
+                        help='path to directories or files to skip. This option'
+                             ' is repeatable.',
+                        default=[])
+    parser.add_argument('-c', dest='config_file',
+                        help='Specify configuration file to use.  '
+                             'Defaults to ".ansible-lint"')
+    parser.add_argument('--version', action='version', version='%(prog)s ' + __version__)
+    parser.add_argument('playbook', nargs='*')
+    options = parser.parse_args()
```

Comment:
This guide https://docs.python.org/3.9/library/argparse.html#upgrading-optparse-code says that it should be
```suggestion
    args = parser.parse_args()
```

So why did you keep options? OTOH it probably contains both args and options in a single var instead of two vars.